resources/profiles/Voron/machine/Voron 0.1 0.25 nozzle.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "machine",
+  "setting_id": "GM002",
+  "name": "Voron 0.1 0.4 nozzle",
+  "from": "system",
+  "instantiation": "true",
+  "inherits": "fdm_klipper_common",
+  "printer_model": "Voron 0.1",
+  "default_print_profile": "0.10mm Standard 0.25 nozzle @Voron",
+  "nozzle_diameter": [
+    "0.4"
```

Comment:
the diameter is wrong.
Same for other profiles

Reply:
Fixed